src/main/java/seedu/module/Module.java

```diff
@@ -272,16 +273,18 @@ public static String checkGradeType(String gradeToCheck) throws UniModsException
         for (int i = 0; i < GRADES.length; i++) {
             if (gradeToCheck.equals(GRADES[i])) {
                 gradeType = TextUi.GRADED;
+                System.out.println("1");
                 return gradeType;
             }
         }
         for (int i = 0; i < SU_GRADES.length; i++) {
             if (gradeToCheck.equals(SU_GRADES[i])) {
                 gradeType = TextUi.UNGRADED;
+                System.out.println("2");
                 return gradeType;
             }
         }
+        System.out.println("3");
```

Comment:
Magic Number

Reply:
Resolved and updated the PR. You can merge it. Thanks